src/explore-education-statistics-admin/src/pages/admin-dashboard/components/MethodologySummary.tsx

```diff
@@ -375,6 +378,18 @@ const MethodologySummary = ({
           <p>Are you sure you want to remove this adopted methodology?</p>
         </ModalConfirm>
       )}
+      <ModalConfirm
+        open={removeExternalMethodology}
+        title="Remove external methodology"
+        onConfirm={async () => {
+          await handleRemoveExternalMethodology();
+          setRemoveExternalMethodology(false);
```

Comment:
Rather than toggle this here, could do it in the `handleRemoveExternalMethodology` method? This would make the modal prop more succinct:

```jsx
onConfirm={handleRemoveExternalMethodology}
```